managementportal-client/src/main/kotlin/org/radarbase/management/client/MPUser.kt

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2021. The Hyve
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ * See the file LICENSE in the root of this repository.
+ */
+
+package org.radarbase.management.client
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import java.time.ZonedDateTime
+
+data class MPUser(
+    @JsonProperty("login")
```

Comment:
In `UserDTO` class I see two separate fields — `String login` and `Long id`. Shouldn't fields of `MPUser` correspond to those of `UserDTO`?

Reply:
Indeed. Well for purposes of external clients, the ID of the user is meaningless: it's an internal database identifier not used elsewhere. For all intents and purposes, the login acts as an ID instead. By converting the name, the intent of the login field as identifier becomes clearer in my opinion.